contents/staticAnalysis/FindBugs.md

```diff
@@ -1,16 +1,71 @@
 # FindBugs
 
-Author: [Xiao Pu](https://nus-oss.github.io/cs3281-website/students/AY1617S2/xiaoPu/xiaoPu-Resume.html)
+Author: [Xiao Pu](https://nus-oss.github.io/cs3281-website/students/AY1617S2/xiaoPu/xiaoPu-Resume.html), [Shradheya Thakre](https://github.com/tshradheya)
 
 ## Overview
 
-FindBugs is a static analysis tool to find "bugs" in **Java** programme. It looks for "bug patterns" in the code and signals possible violations. For example, the bug "null pointer dereference" has the pattern — A programme declares a non-nullable variable but assigns `null` to the variable somewhere and uses it later.
+FindBugs is a static analysis tool to find "bugs" in **Java** programme. It looks for "bug patterns" in the code and signals possible violations. Potential errors are classified in four ranks:
+1. scariest,
+1. scary,
+1. troubling,
+1. of concern.
+
+This is a hint to the developer about their possible impact or severity For example, the bug "null pointer dereference" has the pattern — A programme declares a non-nullable variable but assigns `null` to the variable somewhere and uses it later.
```

Comment:
1. You missed a fullstop after "severity"
2. Also, it would be good to highlight the pattern in a different format from the text. Perhaps put them in quotes or something to differentiate the text from the message.